debezium-server/debezium-server-pubsub/src/main/java/io/debezium/server/pubsub/PubSubChangeConsumer.java

```diff
@@ -85,10 +129,34 @@ void connect() {
             return;
         }
 
+        BatchingSettings.Builder batchingSettings = BatchingSettings.newBuilder()
+                .setDelayThreshold(Duration.ofMillis(maxDelayThresholdMs))
```

Comment:
Wow, that's really unfortunate that they expose the 310 stuff in this signature :(